scripts/prep/cam/exevs_cam_severe_prep.sh

```diff
@@ -54,6 +54,7 @@ fi
 # Send missing data alert if needed
 if [ $data_missing ]; then
 
+   echo "WARNING: File $DCOMINspc/${OTLK_DATE}/validation_data/weather/spc/day*otlk_{OTLK_DATE}*.zip is missing" >> mailmsg
```

Comment:
I think the redirect of this echo statement to mailmsg should be removed.

Reply:
@ShelleyMelchior-NOAA Thanks for catching this.  Removed!